tests/xtro-sharpie/common-CloudKit.ignore

```diff
@@ -9,6 +9,7 @@
 ## Not intended to be used directly by client code
 !missing-protocol! CKRecordKeyValueSetting not bound
 !missing-protocol-conformance! CKRecord should conform to CKRecordKeyValueSetting (defined in 'CKRecordKeyValueSettingConformance' category)
+!missing-selector! CKRecord::encryptedValues not bound
```

Comment:
Not to be added due to the same reason, also intro says the type cannot be loaded.